magpie/api/exception.py

```diff
@@ -373,31 +369,31 @@ def format_content_json_str(httpCode, detail, content, contentType):
     """
     json_body = {}
     try:
-        content[u"code"] = httpCode
+        content[u"code"] = http_code
         content[u"detail"] = detail
-        content[u"type"] = contentType
+        content[u"type"] = content_type
         json_body = json.dumps(content)
-    except Exception as e:
-        msg = "Dumping json content '{!s}' resulted in exception '{!r}'.".format(content, e)
-        raise_http(httpError=HTTPInternalServerError, detail=msg,
-                   contentType=CONTENT_TYPE_JSON,
+    except Exception as exc:
+        msg = "Dumping json content '{!s}' resulted in exception '{!r}'.".format(content, exc)
+        raise_http(http_error=HTTPInternalServerError, detail=msg,
+                   content_type=CONTENT_TYPE_JSON,
                    content={u"traceback": repr(exc_info()),
-                            u"exception": repr(e),
+                            u"exception": repr(exc),
                             u"caller": {u"content": repr(content),  # raw string to avoid recursive json.dumps error
                                         u"detail": detail,
-                                        u"code": httpCode,
-                                        u"type": contentType}})
+                                        u"code": http_code,
+                                        u"type": content_type}})
     return json_body
 
 
 # noinspection PyPep8Naming
-def generate_response_http_format(httpClass, httpKWArgs, jsonContent, outputType=CONTENT_TYPE_PLAIN):
+def generate_response_http_format(httpClass, http_kwargs, jsonContent, outputType=CONTENT_TYPE_PLAIN):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "outputType" doesn't conform to snake_case naming style](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)